var/spack/repos/builtin/packages/aocl-sparse/package.py

```diff
@@ -17,29 +19,24 @@ class AoclSparse(CMakePackage):
     LICENSING INFORMATION: By downloading, installing and using this software,
     you agree to the terms and conditions of the AMD AOCL-Sparse license agreement.
     You may obtain a copy of this license agreement from
-    https://www.amd.com/en/developer/aocl/sparse/sparse-libraries-4-0-eula.html
-    https://www.amd.com/en/developer/aocl/sparse/sparse-libraries-eula.html
+    https://www.amd.com/en/developer/aocl/sparse/eula/sparse-libraries-4-1-eula.html
+    https://www.amd.com/en/developer/aocl/sparse/eula/sparse-libraries-eula.html
     """
 
     _name = "aocl-sparse"
     homepage = "https://www.amd.com/en/developer/aocl/sparse.html"
     url = "https://github.com/amd/aocl-sparse/archive/3.0.tar.gz"
-    git = "https://github.com/amd/aocl-sparse.git"
+    git = "https://github.com/amd/aocl-sparse"
 
     maintainers("amd-toolchain-support")
 
+    version("4.1", sha256="68524e441fdc7bb923333b98151005bed39154d9f4b5e8310b5c37de1d69c2c3")
```

Comment:
How is this checksum calculated?  

I'm getting `35ef437210bc25fdd802b462eaca830bfd928f962569b91b592f2866033ef2bb` from `spack checksum aocl-sparse`.